src/VisualStudio/Core/Test/DebuggerIntelliSense/CSharpDebuggerIntellisenseTests.vb

```diff
@@ -234,16 +226,15 @@ Namespace Microsoft.VisualStudio.LanguageServices.UnitTests.DebuggerIntelliSense
             Using state = TestState.CreateCSharpTestState(text, False)
                 state.SendTypeChars("variable")
                 Await state.WaitForAsynchronousOperationsAsync()
-                Await state.AssertCompletionItemsContainNone("variable1")
-                Await state.AssertCompletionItemsContainAll("variable2")
+                Await state.AssertCompletionItemsDoNotContainAny({"variable1"})
+                Await state.AssertCompletionItemsContainAll({"variable2"})
```

Comment:
❔ Did we lose a `ParamArray`, or is there some other reason why the braces are now required?

Reply:
Restored ParamArray.

---
In reply to: [325262491](https://github.com/dotnet/roslyn/pull/38569#discussion_r325262491) [](ancestors = 325262491)